README.md

```diff
@@ -1 +0,0 @@
-# protractor-workshop-2017
```

Comment:
don't remove the README file, it is a good practice to have it, later on you can add more information to it with a description of the project, steps to executed, tools, etc